src/concourse/pipelines/infrastructure/ocw_studio_db_replication/pipeline.py

```diff
@@ -0,0 +1,84 @@
+from concourse.lib.constants import REGISTRY_IMAGE
+from concourse.lib.models.pipeline import (
+    AnonymousResource,
+    Command,
+    Identifier,
+    Input,
+    Job,
+    Output,
+    Pipeline,
+    Platform,
+    RegistryImage,
+    TaskConfig,
+    TaskStep,
+)
+
+postgres_resource = AnonymousResource(
+    type=REGISTRY_IMAGE, source=RegistryImage(repository="postgres", tag="12")
+)
+
+
+def db_replication_pipeline() -> Pipeline:
+    db_replication_job = Job(
+        name="restore-db",
+        build_log_retention={"builds": 10},
+        plan=[
+            TaskStep(
+                task=Identifier("run-pg-dump"),
+                privileged=False,
+                config=TaskConfig(
+                    platform=Platform.linux,
+                    image_resource=postgres_resource,
+                    outputs=[Output(name="dump")],
+                    params={
+                        "PGHOST": "((source.host))",
```

Comment:
Is this credential being written to a Vault path scoped to the pipeline? Otherwise I'd suggest that the var name be changed to something more descriptive/distinctive. (same for the `destination` name below)

Reply:
So based on where this pipeline is run (team + name), the secret path is actually @ 
`ocw/ocw-studio-db-replication/source` and `
ocw/ocw-studio-db-replication/destination` which is kind of puzzling where that middle part comes from. It is obviously the pipeline name but I don't feel like I've seen it before.

Reply:
This is inferred from the pipeline name at the time `set-pipeline` is run based on the command printed at the end: `fly -t qa-ocw set-pipeline -p ocw-studio-db-replication -c definition.json`. It could be clearer that the pipeline name is actually important, but either way if it's not changed then it should take vars set on the pipeline name as described [here](https://concourse-ci.org/vault-credential-manager.html#vault-credential-lookup-rules) with `PIPELINE_NAME` being the name of the pipeline when pushed up. Do y'all think I should still be more explicit and have it not depend on the pipeline name?

Reply:
I think it is probably okay how it is. It makes sense to me now.